### PR TITLE
Added Sundial Pagerduty Integration

### PR DESCRIPTION
**Pull request by nemo83**
Addee Sundial Pagerduty Integration, draft

### CODE REVIEW

File: _posts/2017-02-03-sundial-pagerduty-integration.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Sundial Pagerduty Integration
```
Comment (jkenny) [inline _posts/2017-02-03-sundial-pagerduty-integration.md]: PagerDuty should be camel case.

File: _posts/2017-02-03-sundial-pagerduty-integration.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+# Sundial
+A few months ago, Gilt Tech announced [Sundial](). Sundial is an open source batch job scheduler for Amazon ECS.
+Over the course of the last few months, Sundial has seen a significant adoption both inside and outside of Gilt.
+
+Until Sundial v0.0.10, emailing was the only way of notifying job failures.
+
+At the beginning when the number of jobs running on Sundial was small (and so were failures!), it was fairly easy to spot emails of failed jobs and act accordingly.
+
+Lately though, in the Personalization Team, Sundial schedules more than XYZ job executions per day and it's easy to imagine the amount of noise in our inbox generated by job notifications.
```
Comment (jkenny) [inline _posts/2017-02-03-sundial-pagerduty-integration.md]: Replace XYZ with a real number.

File: _posts/2017-02-03-sundial-pagerduty-integration.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ 
+Beside the noise, it has happened more than once that failure of critical jobs went unnoticed. This was of course unacceptable.
+ 
+Since [Pagerduty](https://www.pagerduty.com/) is the _de facto_ standard in Gilt when it comes to _on call procedures_ and since Pagerduty offers a nice and reliable events [API](https://v2.developer.pagerduty.com/docs/events-api), we've redesigned the notification mechanism 
+and integrated Pagerduty in Sundial.
```
Comment (jkenny) [inline _posts/2017-02-03-sundial-pagerduty-integration.md]: "integrated ... with"

File: _posts/2017-02-03-sundial-pagerduty-integration.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  * _on\_state\_change_, Notify when a process goes from succeeding to failing and vice versa
+  * _on\_state\_change\_and\_failures_, Notify when going from failing to succeeded and on each failure
+  * _never_
+* _my\_pd\_service\_key_ is the key obtained in the _Service Page_ in Pagerduty
+* _num\_consecutive\_failures_ is the number of consecutive failures after which Sundial will trigger an alert in Pagerduty
+
+Please note that the `subscriptions` object in the Process Definition json has been deprecated, so if you've already adopted Sundial and want to start using the new `notifications`, you will have to 
+ update your json accordingly. 
+
+More details can be found in the [Sundial v.0.0.10 release page](https://github.com/gilt/sundial/releases/tag/v0.0.10)
```
Comment (jkenny) [inline _posts/2017-02-03-sundial-pagerduty-integration.md]: Is Sundial mature enough for v1 yet?

Comment (nemo83) [inline _posts/2017-02-03-sundial-pagerduty-integration.md]: I'd say it is. @kevinoriordan what do you think?

Comment (kevinoriordan) [inline _posts/2017-02-03-sundial-pagerduty-integration.md]: Code itself would be ready I think. Documentation could be improved a lot. It's already been open sourced anyway.

File: _posts/2017-02-03-sundial-pagerduty-integration.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+- pagerduty
+---
+
+# Sundial
+A few months ago, Gilt Tech announced [Sundial](https://github.com/gilt/sundial). Sundial is an open source batch job scheduler for Amazon ECS.
+Over the course of the last few months, Sundial has seen a significant adoption both inside and outside of Gilt.
+
+Until Sundial v0.0.10, emailing was the only way of notifying job failures.
+
+At the beginning when the number of jobs running on Sundial was small (and so were failures!), it was fairly easy to spot emails of failed jobs and act accordingly.
```
Comment (JasonTam) [inline _posts/2017-02-03-sundial-pagerduty-integration.md]: Were the failures small, or _the number of_ failures small? (just some parallelism)

Comment (nemo83) [inline _posts/2017-02-03-sundial-pagerduty-integration.md]: will correct into *and so were the number of failures*

**Pull request merged**